torchx/Makefile

```diff
@@ -34,6 +37,10 @@ check_libtorch:
 
 $(TORCHX_SO): c_src/torchx.cpp c_src/nx_nif_utils.hpp
 	mkdir -p $(PRIV_DIR)
-	ln -sf $(abspath $(LIBTORCH_LINK_PATH)) $(LIBTORCH_LIB_DIR)
-	$(CXX) $(CFLAGS) c_src/torchx.cpp -o $(TORCHX_SO) $(LDFLAGS)
-	$(POST_INSTALL)
+	mkdir -p $(CMAKE_BUILD_DIR)
+	cd $(CMAKE_BUILD_DIR) && \
+		cmake -DCMAKE_PREFIX_PATH=$(LIBTORCH_DIR) -DC_SRC=$(C_SRC) \
```

Comment:
@josevalim please check if this works on mac, or if we need to add the `ln -s` back